src/style.css

```diff
@@ -1 +1,37 @@
-/* add styles here */
+.page {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: url('./images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  padding: 1px;
```

Comment:
Consider using only top or bottom margin/padding to avoid potential margin collapse. The checklist suggests being consistent with your margins.